.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Help us fix things that are broken
+title: ''
+labels: ''
+assignees: ''
+
```

Comment:
Pardon my friday-brain but what is this front-matter for? How does it work and what's it helping?